src/trio/_core/_run.py

```diff
@@ -2724,16 +2747,17 @@ def unrolled_run(
                     #   https://github.com/python/cpython/issues/108668
                     # So now we send in the Outcome object and unwrap it on the
                     # other side.
-                    msg = task.context.run(next_send_fn, next_send)
+                    msg = task.context.run(run_with, next_send_fn, next_send)
                 except StopIteration as stop_iteration:
                     final_outcome = Value(stop_iteration.value)
                 except BaseException as task_exc:
                     # Store for later, removing uninteresting top frames: 1
                     # frame we always remove, because it's this function
+                    # another is the run_with
```

Comment:
"because it's this function another is the run_with catching it" does not make sense

Reply:
this will be removed when switching to the contextvar